k8s/apis/solution/v1/solution_webhook.go

```diff
@@ -33,6 +33,11 @@ func (r *Solution) SetupWebhookWithManager(mgr ctrl.Manager) error {
 		solution := rawObj.(*Solution)
 		return []string{solution.Spec.DisplayName}
 	})
+	mgr.GetFieldIndexer().IndexField(context.Background(), &Solution{}, ".spec.rootResource", func(rawObj client.Object) []string {
```

Comment:
Is there any scenario that users only configure ownerReference but forget spec.rootResource?

Reply:
No such scenario. `rootResource` is a required filed and checked in webhook.